CHANGELOG.md

```diff
@@ -26,6 +26,7 @@ Read our [guidelines for writing a good changelog entry](https://github.com/biom
 #### Bug fixes
 
 - Fix [#182](https://github.com/biomejs/biome/issues/182), making [useLiteralKeys](https://biomejs.dev/linter/rules/use-literal-keys/) retains optional chaining.
+- Fix [#137](https://github.com/biomejs/biome/issues/137). Now treats `TS_EXTERNAL_MODULE_DECLARATION` as a new scope in the semantic model.
```

Comment:
We should report that we fixed some false positives in `noRedeclare`. Our users don't really know/understand what `TS_EXTERNAL_MODULE_DECLARATION` really mean 😊